hail/src/main/scala/is/hail/expr/ir/functions/Functions.scala

```diff
@@ -377,8 +352,8 @@ abstract class RegistryFunctions {
     })
   }
 
-  def registerScalaFunction(mname: String, argTypes: Array[Type], rType: Type, pt: (Type, Seq[PType]) => PType)(cls: Class[_], method: String) {
-    registerCode(mname, argTypes, rType, pt) { case (r, rt, args) =>
+  def registerScalaFunction(mname: String, argTypes: Array[Type], rType: Type, pt: (Type, Seq[PType]) => PType, typeParams: Array[Type] = Array.empty)(cls: Class[_], method: String) {
```

Comment:
not a big deal, but we don't need typeParams here, no registerScalaFunction passes this and registerCode now has a default.

Reply:
good point.